templates/api.hbs

```diff
@@ -32,6 +39,9 @@ export default function(app: Express, impl: t.{{className name}}Api) {
 						{{#unless schema}}
 						return undefined
 						{{else}}
+						{{#if (isContentSupportsMultipart .)}}
```

Comment:
We have other tests in here that are inline in the `.hbs` file... I feel like we could create `isContentMultipart` and `isContentJson` handlers and put them up into a very base class generator and change other generator templates to use those rather than this smorgasbord of different ways of checking the content type.